openapi/tests/openapi_integration/test_collection_update.py

```diff
@@ -26,6 +26,12 @@ def test_collection_update():
                         "m": 32,
                         "ef_construct": 123,
                     },
+                    "quantization_config": {
```

Comment:
This test just makes sure we do not get an error.
Maybe we should do an extra GET to validate that the config was changed.